scipy/io/mmio.py

```diff
@@ -192,12 +200,38 @@ def writer():
     #---------------------------------------------------------------------------
     @classmethod
     def info(self, source):
-        source, close_it = self._open(source)
+        """
+        Queries the contents of the Matrix Market file 'source' to
+        extract size and storage information.
+    
+        Parameters
+        ----------
+
+        source : str or file-like
+            Matrix Market filename (extension .mtx) or open file object
+    
+        Returns
+        -------
+    
+        rows,cols : int
```

Comment:
I think these should be two separate entries, because it's ambiguous whether (rows, cols) is a tuple or two separate values.